### PR TITLE
Docker Desktop 4.22.0 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
### Proposed changes

Docker Desktop 4.22.0 release ntoes

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,17 +24,42 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+{% include release-date.html date="2023-07-27" %}
 
-{% include desktop-install.md all=true version="4.21.1" build_path="/" %}
+{% include desktop-install.md all=true version="4.22.0" build_path="/" %}
 
 ### Upgrades
-- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+- [Buildx v0.11.2](https://github.com/docker/buildx/releases/tag/v0.11.2)
+- [Compose v2.20.0](https://github.com/docker/compose/releases/tag/v2.20.0)
```
Comment (milas) [inline desktop/release-notes.md:36]:
```suggestion
- [Compose v2.20.2](https://github.com/docker/compose/releases/tag/v2.20.2)
```

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,17 +24,42 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+#### For all platforms
+- Extended Settings-Management features to allow IT/admins to disable Docker-Desktop "extensions".
+- Fixed a bug where enabling Kubernetes from the UI failed when the system is paused.
+- Fixed a bug where enabling Wasm from the UI failed when the system is paused.
+- Bindmounts are shown in the **Container** view.
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
```
Comment (djs55) [inline desktop/release-notes.md:50]:
```suggestion
- Docker Desktop will use much less CPU and memory when not running containers.
```

File: desktop/release-notes.md [hunk 3]
```diff
@@ -24,17 +24,42 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
-- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+- [Buildx v0.11.2](https://github.com/docker/buildx/releases/tag/v0.11.2)
+- [Compose v2.20.0](https://github.com/docker/compose/releases/tag/v2.20.0)
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- Extended Settings-Management features to allow IT/admins to disable Docker-Desktop "extensions".
```
Comment (mikesir87) [inline desktop/release-notes.md:42]:
```suggestion
- Extended Settings Management features to allow IT/admins to disable Docker Desktop "extensions".
```

File: desktop/release-notes.md [hunk 4]
```diff
@@ -24,17 +24,42 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixed a bug where enabling Kubernetes from the UI failed when the system is paused.
+- Fixed a bug where enabling Wasm from the UI failed when the system is paused.
+- Bindmounts are shown in the **Container** view.
```
Comment (mikesir87) [inline desktop/release-notes.md:45]:
```suggestion
- Bind mounts are shown in the **Container** view.
```

File: desktop/release-notes.md (hunk 2 above)
Comment (aevesdocker) [reply desktop/release-notes.md:53]: @doringeman I've added that it's an experimental feature. Not convinced we should add the rest of the note. It works well in a blog post, but not so well in release notes.

Comment (doringeman) [reply desktop/release-notes.md:53]: Can we use `It massively reduces`?
I've heard from Chris that the same expression will be used for the blog. 😁 

Moreover, it's enabled for Windows Hyper-V (not WSL2) as well.

Comment (aevesdocker) [reply desktop/release-notes.md:53]: Noted! Will change now

File: desktop/release-notes.md [hunk 5]
```diff
@@ -24,17 +24,45 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 
-{% include desktop-install.md all=true version="4.21.1" build_path="/" %}
+{% include desktop-install.md all=true version="4.22.0" build_path="/" %}
 
 ### Upgrades
-- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+- [Buildx v0.11.2](https://github.com/docker/buildx/releases/tag/v0.11.2)
+- [Compose v2.20.2](https://github.com/docker/compose/releases/tag/v2.20.2)
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
```
Comment (aevesdocker) [inline desktop/release-notes.md:37]:
```suggestion
- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)

```

File: desktop/release-notes.md [hunk 6]
```diff
@@ -24,17 +24,45 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Extended Settings Management features to allow IT/admins to disable Docker Desktop "extensions".
+- Fixed a bug where turning on Kubernetes from the UI failed when the system was paused.
+- Fixed a bug where turning on Wasm from the UI failed when the system was paused.
+- Bind mounts are shown in the **Container** view.
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
+Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (aevesdocker) [inline desktop/release-notes.md:54]:
```suggestion
Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.

```

File: desktop/release-notes.md [hunk 7]
```diff
@@ -24,17 +24,45 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+#### For Windows
+
+- Fixed a bug where the self-diagnose tool showed a false-positive failure when vpnkit is expected to be not running. Fixes [docker/for-win#13479](https://github.com/docker/for-win/issues/13479).
+- Fixed a bug where an invalid regular expression in the search bar caused an error. Fixes [docker/for-win#13592](https://github.com/docker/for-win/issues/13592).
+
```
Comment (aevesdocker) [inline desktop/release-notes.md:59]:
```suggestion
Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.

```

File: desktop/release-notes.md [hunk 8]
```diff
@@ -24,17 +24,45 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- [Resource Usage](use-desktop/container.md) has moved from experimental to GA.
+- You can now split large Compose projects into multiple sub-projects with [`include`](../compose/multiple-compose-files/include.md).
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- Extended Settings Management features to allow IT/admins to disable Docker Desktop "extensions".
+- Fixed a bug where turning on Kubernetes from the UI failed when the system was paused.
+- Fixed a bug where turning on Wasm from the UI failed when the system was paused.
+- Bind mounts are shown in the **Container** view.
```
Comment (aevesdocker) [inline desktop/release-notes.md:48]:
```suggestion
- Bind mounts are now shown when you [inspect a container](use-desktop/container.md).
```

File: desktop/release-notes.md [hunk 9]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Extended Settings Management features to allow IT/admins to disable Docker Desktop "extensions".
+- Fixed a bug where turning on Kubernetes from the UI failed when the system was paused.
+- Fixed a bug where turning on Wasm from the UI failed when the system was paused.
+- Bind mounts are now shown when you [inspect a container](use-desktop/container.md).
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
+Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (aevesdocker) [inline desktop/release-notes.md:55]:
```suggestion
- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```

File: desktop/release-notes.md [hunk 10]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+#### For Windows
+
+- Fixed a bug where the self-diagnose tool showed a false-positive failure when vpnkit is expected to be not running. Fixes [docker/for-win#13479](https://github.com/docker/for-win/issues/13479).
+- Fixed a bug where an invalid regular expression in the search bar caused an error. Fixes [docker/for-win#13592](https://github.com/docker/for-win/issues/13592).
+Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (aevesdocker) [inline desktop/release-notes.md:61]:
```suggestion
- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```

File: desktop/release-notes.md [hunk 11]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
+
+### New
+- [Resource Usage](use-desktop/container.md) has moved from experimental to GA.
+- You can now split large Compose projects into multiple sub-projects with [`include`](../compose/multiple-compose-files/include.md).
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- Extended Settings Management features to allow IT/admins to disable Docker Desktop "extensions".
```
Comment (aevesdocker) [inline desktop/release-notes.md:46]:
```suggestion
- [Settings Management](hardened-desktop/settings-management.md) now lets you turn off Docker Extensions for your organisation.
```

File: desktop/release-notes.md [hunk 12]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
+- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
+
+#### For Windows
+
+- Fixed a bug where the self-diagnose tool showed a false-positive failure when vpnkit is expected to be not running. Fixes [docker/for-win#13479](https://github.com/docker/for-win/issues/13479).
+- Fixed a bug where an invalid regular expression in the search bar caused an error. Fixes [docker/for-win#13592](https://github.com/docker/for-win/issues/13592).
+- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (aevesdocker) [inline desktop/release-notes.md:61]:
```suggestion
- Resource Saver is available as an experimental feature for Windows Hyper-V. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```

File: desktop/release-notes.md [hunk 13]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- [Docker Engine v24.0.5](https://docs.docker.com/engine/release-notes/24.0/#2405)
+
+### New
+- [Resource Usage](use-desktop/container.md) has moved from experimental to GA.
+- You can now split large Compose projects into multiple sub-projects with [`include`](../compose/multiple-compose-files/include.md).
 
 ### Bug fixes and enhancements
 
+#### For all platforms
+- [Settings Management](hardened-desktop/settings-management.md) now lets you turn off Docker Extensions for your organisation.
```
Comment (aevesdocker) [inline desktop/release-notes.md:46]:
```suggestion
- [Settings Management](hardened-desktop/settings-management/index.md) now lets you turn off Docker Extensions for your organisation.
```

File: desktop/release-notes.md [hunk 14]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- [Settings Management](hardened-desktop/settings-management/index.md) now lets you turn off Docker Extensions for your organisation.
+- Fixed a bug where turning on Kubernetes from the UI failed when the system was paused.
+- Fixed a bug where turning on Wasm from the UI failed when the system was paused.
+- Bind mounts are now shown when you [inspect a container](use-desktop/container.md).
+- You can now download Wasm runtimes when the containerd image store is enabled.
+
+#### For Mac
+
+- Fixed a bug that prevented Docker Desktop from starting. [docker/for-mac#6890](https://github.com/docker/for-mac/issues/6890)
+- Resource Saver is available as an experimental feature. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (christophermclellan) [inline desktop/release-notes.md:55]: @aevesdocker @doringeman sorry for late feedback. i wouldn't lead with the fact that this is experimental, it's being rolled out to all users by default.

can we please adjust to:

"Resource Saver is now available on Mac and Windows, optimising Docker Desktop's usage of your system resources when no containers are running."

Comment (aevesdocker) [reply desktop/release-notes.md:55]: I think we still need to highlight that you'll only see this feature if you have enabled access to experimental features. The current phrasing matches how we've introduced other experimental features in release notes, but how about 
"Resource Saver is now available on Mac and Windows. It optimises Docker Desktop's usage of your system resources when no containers are running. To access this feature, make sure you have enabled access to experimental features."

Comment (doringeman) [reply desktop/release-notes.md:55]: I agree!

And use "It **massively** reduces"? 😁  https://github.com/docker/docs/pull/17821#discussion_r1282978170

Comment (christophermclellan) [reply desktop/release-notes.md:55]: My only concern here is that we're labelling this as 'experimental' but it's going to be default enabled. But I guess that is the case for all experimental features (e.g. they're _all_ default-enabled), so my concern is perhaps invalid.

Happy to go with the wording you propose @aevesdocker , and also take your steer on whether 'massively' is the right language for the Release Notes. Though I appreciate the marketing @doringeman 😂

Comment (aevesdocker) [inline desktop/release-notes.md:55]:
```suggestion
- Resource Saver is now available on Mac. It optimises Docker Desktop's usage of your system resources when no containers are running. To access this feature, make sure you have [turned on access to experimental features](settings/mac.md) in settings.
```

File: desktop/release-notes.md [hunk 15]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+#### For Windows
+
+- Fixed a bug where the self-diagnose tool showed a false-positive failure when vpnkit is expected to be not running. Fixes [docker/for-win#13479](https://github.com/docker/for-win/issues/13479).
+- Fixed a bug where an invalid regular expression in the search bar caused an error. Fixes [docker/for-win#13592](https://github.com/docker/for-win/issues/13592).
+- Resource Saver is available as an experimental feature for Windows Hyper-V. It reduces the amount of CPU and memory used by Docker Desktop when not running containers.
```
Comment (aevesdocker) [inline desktop/release-notes.md:61]:
```suggestion
- Resource Saver is now available on Windows Hyper-V. It optimises Docker Desktop's usage of your system resources when no containers are running. To access this feature, make sure you have [turned on access to experimental features](settings/windows.md) in settings.
```

File: desktop/release-notes.md [hunk 16]
```diff
@@ -24,17 +24,48 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.21.1
+## 4.22.0
 
-{% include release-date.html date="2023-07-03" %}
+{% include release-date.html date="2023-07-27" %}
```
Comment (aevesdocker) [inline desktop/release-notes.md:29]:
```suggestion
{% include release-date.html date="2023-08-03" %}
```

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**